.github/workflows/ci_cron.yml

```diff
@@ -59,10 +59,16 @@ jobs:
       artifact-path: |
         results.xml
       envs: |
+        - linux: check-docs-links
+        - macos: py311-xdist
+          pytest-results-summary: true
         - macos: py312-xdist
           pytest-results-summary: true
-        - linux: py311-xdist
+        - linux: py3-stdevdeps-xdist
+          pytest-results-summary: true
+        - macos: py3-stdevdeps-xdist
```

Comment:
Didn't @braingram say we should just merge stdevdeps into devdeps?

Reply:
From offline discussion with @zacharyburnett he convinced me this is a large enough change to be put in a follow-up PR. @pllim does that work for you?

Reply:
Sure. Just want to make sure we not overlooking it. Thanks!